public/ng-views/maps.html

```diff
@@ -550,78 +621,109 @@ <h3 class="noTopMargin noBottomMargin">
+          <img class="stationIcons" src="public/imgs/brt_black_yellow.svg">
           BRT
         </label>
         <label class="btn btn-success" ng-click="saveAlt=true; createVar=false">
-          <i class="fa fa-cloud-upload"></i> Save Variation
+          <i class="fa fa-cloud-upload"></i> Save
         </label>
       </div>
     </div>
   </div>
 
 </div>
+<!--END of ROUTE EDITOR MODULE--> 
 
+<div class="controlModules">
 
 
-<div class="basecol">
+<!--POI MARKER ACTIVATION --> 
+
+<div class="basecol_POI col-md-3 "> 
+<!-- <h5 style="margin-bottom: 10px"> PLACES </h5> -->
+  <div class="fa-stack" ng-click="targetPOIUsers(); poiUserDots = !poiUserDots" style="margin-bottom: 8px">
+    <i class="fa fa-circle fa-stack-2x"></i>
+    <i class="fa fa-users fa-stack-1x fa-inverse destinationIcon iconContainer"></i>
+  </div>
+  <br>
+  <span ng-show="poiUserDots">
+    <span class="fa-stack userDot" 
+          ng-repeat="user in poiUsers track by $index" 
+          ng-click="targetPOIUsers(user.name)" 
+          ng-class="user.name == currentPOIUser ? '' : 'dot-unselected'"
+          style="color:#{{ user.color }}">
+      <i class="fa fa-circle fa-stack-2x"></i>
+      <i class="fa fa-stack-1x fa-inverse">
+        <small>{{ user.name }}</small>
+      </i>
+    </span>
+  <span>
+
+</div>
+<!--End of POI MARKER ACTIVATION --> 
+
+
+<!--DESTINATION & TIME FILTER --> 
+
+<div class="basecol_filter col-md-3 ">
 
   <div class="progress" ng-show="loadProgress.vis">
-    <div class="progress-bar progress-bar-info progress-bar-striped active"
+    <div class="progress-bar progress-bar-info progress-bar-striped active" 
           role="progressbar"  id="isosProgressBar"
           style="width:{{loadProgress.val}}%">
       {{loadProgress.val}}%
     </div>
   </div>
+  <h5 style="margin-bottom: -7px">
+    TRAVEL TIME: 
+    <span ng-click="toggleShowVectorIsos()">
+            {{ vectorIsos.val*5 | minuteConverter }}
+            <span class="badge">
+            <i class="fa fa-eye" ng-show="showVectorIsosOn"></i>
+            <i class="fa fa-eye-slash" ng-show="!showVectorIsosOn"></i>
+            </span>
+    </span>
+  <h5>
 
-  <div class="row" ng-show="!scenarioCompare && !loadProgress.vis">
-    <div class="col-xs-12 body pull-right isochroneSlider">
-      <span style="width:25%;display:inline-block" class="pull-left">
-        <h6>
-          Travel Time Bounds
-        </h6>
-      </span>
+<div class="isochroneSlider">
+
+<!-- <div class="col-xs-12 body pull-right isochroneSlider">
+ -->
       <span class="text-center" ng-class="showVectorIsosOn ? '' : 'unselected'">
-        <i class="fa fa-minus-square"
+        <i class="fa fa-minus-square" 
             ng-click="vectorTimeVal_subtract(); showVectorIsos(300*vectorIsos.val)"></i>
         <span style="width:50%;display:inline-block">
           <input type="range" min="1" max="24" class="isosRangeBar" style="top:10px; margin-top:8px; margin-bottom:-4px"
```

Comment:
This range is broken and unusable as it now stands. It can't just sit in a div - it needs to be responsive to the div. 

![image](https://cloud.githubusercontent.com/assets/6053396/8777318/57b66c5a-2ec5-11e5-92a5-f202ee5bda10.png)

See the above image.